components/utils.py

```diff
@@ -1158,7 +1181,18 @@ def _load_node_props(service_name):
 
     def _get_local_file_path(self, service_name, resource_name):
         base_service_res_dir = self.get_resources_dir(service_name)
-        dest_file_path = os.path.join(base_service_res_dir, resource_name)
+
+        if (resource_name.startswith('cloudify')
+            or resource_name.find('-agent_') != -1) \
```

Comment:
`'-agent_' not in resource_name` is nicer than using `.find` (with its -1 weirdness)